man/journald.conf.xml

```diff
@@ -103,6 +103,10 @@
         <para>Note that per-user journal files are not supported unless persistent storage is enabled, thus
         making <command>journalctl --user</command> unavailable.</para>
 
+        <para>The storage to use can also be specified via the <literal>journal.storage</literal>
```

Comment:
If you touch this, please add a proper `Credentials` section to `systemd-journald.service(8)`, with version info and such

Reply:
(and document `journal.forward_to_socket` there, too)